### DISCUSSION TITLE
Why we do not have the rpms/deb for the milvus cluster ?

### DISCUSSION THREAD

**Discussion by Balaji5007** [milvus-io/milvus#30841] [Q&A and General discussion] [answered]
Why we do not have the rpms/deb for the milvus cluster ?  
I want to create a milvus cluster in the ubuntu linux servers, i see that we need a k8's to build the milvus clusters, why can't we create a cluster with the rpms/deb  ?

**Comment by yhmo** [comment, marked as answer]
Standalone is a single process. You can easily start a single process by systemctl.
Cluster is multi-processes with third-parties(Etcd/Pulsar/Minio). It is difficult to manage these processes by systemctl.

**Comment by Balaji5007** [comment]
Thanks for the response, and again this is forcing us to use the Milvus only through the kubernetes way for the cluster rather give it to the end users with more options in choosing the platform.

**Comment by Balaji5007** [comment]
We are trying to have ecs s3 endpoint instead of a Minio end point in a cluster, hence the complication starts

**Reply by yhmo** [reply]
Milvus uses compatible S3 interfaces for storage, you can config AWS/GCS for milvus:
https://milvus.io/docs/deploy_s3.md
https://milvus.io/docs/object_storage_operator.md
